com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,143 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --filesystem=xdg-config/gtk-3.0:ro
+  - --persist=.vscode-oss
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=LD_LIBRARY_PATH=/app/lib
```

Comment:
This should not be needed

Reply:
indeed, I tested and is not more needed, then I remove `--env=LD_LIBRARY_PATH=/app/lib`